dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexUtilTest.java

```diff
@@ -0,0 +1,199 @@
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is read only
+     * Should:return true
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnTrueIfTheCurrentIndicesAreReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), true);
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(true, readOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index not exists
+     * Should: throw a {@link org.elasticsearch.ElasticsearchStatusException}
+     *
+     * @throws DotDataException
+     */
+    @Test (expected = ElasticsearchStatusException.class)
+    public void whenTheIndexNotExistsShouldThrowException() {
+        final boolean readOnly = ESIndexUtil.isReadOnly("not_Exists");
+        assertEquals(true, readOnly);
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#putReadOnlyToFalse(String...)}
+     * When: The index exists
+     * Should: should set it to read only
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldPutReadonlyInTrue() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            ESIndexUtil.putReadOnlyToFalse(indiciesInfo.getWorking());
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(true, !readOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#putReadOnlyToFalse(String...)}
+     * When: The index not exists
+     * Should: throw a {@link org.elasticsearch.ElasticsearchStatusException}
+     *
+     */
+    @Test(expected = ElasticsearchStatusException.class)
+    public void shouldTryPutReadonlyInTrueAndTheIndexDoesNotExists()  {
+        ESIndexUtil.putReadOnlyToFalse("index_not_exists");
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isAnyCurrentIndicesReadOnly()}
+     * When: If at least one of the current working index if read only
+     * Should: return true
+     *
+     * @throws DotDataException
+     */
+    @Test()
+    public void shouldReturnTrueWhenWorkingIndexIsReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getLive(), false);
+            putReadOnly(indiciesInfo.getWorking(), true);
+
+            final boolean anyCurrentIndicesReadOnly = ESIndexUtil.isAnyCurrentIndicesReadOnly();
+            assertEquals(true, anyCurrentIndicesReadOnly);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use assertTrue(x)/assertFalse(x) instead of assertEquals(true, x)/assertEquals(false, x) or assertEquals(Boolean.TRUE, x)/assertEquals(Boolean.FALSE, x).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)